tests/Tests/500__jupyterhub/test-folder-permissions.robot

```diff
@@ -0,0 +1,65 @@
+*** Settings ***
+Documentation       Test Suite to verify installed library versions
+
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+
+Library             JupyterLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+*** Variables ***
+@{LIST_OF_IMAGES} =    s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                    pytorch    tensorflow
+@{EXPECTED_PERMISSIONS} =       0775    0    1001
+@{FOLDER_TO_CHECK} =            /opt/app-root/lib    /opt/app-root/share
+
+*** Test Cases ***
+Verify Folder Permissions
+    [Documentation]    It checks Access, Gid, Uid of /opt/app-root/lib and /opt/app-root/shares
+    [Tags]    ODS-486
+    ...       Tier2
+    FOR    ${img}    IN    @{LIST_OF_IMAGES}
```

Comment:
`Verify Folder Permissions For Images ` @{LIST_OF_IMAGES}

Reply:
Done